### DISCUSSION TITLE
Announcement text not rendering

### DISCUSSION THREAD

**Discussion by pauljwil** [pradyunsg/furo#437] [Potential Issue] [answered]
I'm trying to present an announcement on the top of every page in my project repo by using the following configuration in the `conf.py` file:

```
html_theme_options = {
    "navigation_depth": 4,
    "light_css_variables": {
        "color-brand-primary": "dark",
    },
    "announcement": "Example announcement",
}
```

(@pradyunsg explained in the Write the Docs Slack that the `color-brand-primary` is not a valid CSS value and that `navigation-depth` is not used in Furo, so I am aware of those two issues.)

The "Example announcement" does not render in my Sphinx build. It does however produce a small amount of additional space between the company name banner at the top of the page and the H1 title where the text of the docs topic begins.

When I inspect the HTML using Chrome browser, hovering over the `announcement` HTML produces a highlighted rectangle over the top-most company name banner, which leads me to believe that the announcement and the banner are competing for the same space.

Do I need to customize the CSS for the announcement in order to get the announcement to present lower on the screen?

Please let me know what further information I might need to share to properly diagnose the problem.

Thanks!

**Comment by pradyunsg** [comment]
Is this HTML/rendered webpage hosted or available somewhere that I can take a look at it?

**Reply by pauljwil** [reply]
Unfortunately, I'm not able to share that info publicly.

**Comment by pradyunsg** [comment, marked as answer]
Closing this out, since I can't really help with this without any way to reproduce this.
